File: Node/Fibonacci/src.js
console.time("Execution time: ");

let step = 50;

recFib(step);

console.timeEnd("Execution time: ");


function recFib(num) {
    if (num <= 2)
        return 1;
    return recFib(num - 1) + recFib(num - 2);
}

File: C#/Fibonacci/src.cs
using System;

namespace Fibonacci
{
    class Program
    {
        public static void Main()
        {
            uint step = 50;
            var watch = System.Diagnostics.Stopwatch.StartNew();

            RecFib(step);

            watch.Stop();
            var elapsedMs = watch.ElapsedMilliseconds;
            Console.WriteLine("Execution time: " + elapsedMs);
            Console.ReadKey();
        }

        private static ulong RecFib(uint num)
        {
            if (num <= 2)
                return 1;
            return (ulong)(RecFib(num - 1) + RecFib(num - 2));
        }
    }
}